pallets/nfts/src/weights.rs

```diff
@@ -818,11 +824,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: `Nfts::Collection` (r:1 w:1)
 	/// Proof: `Nfts::Collection` (`max_values`: None, `max_size`: Some(84), added: 2559, mode: `MaxEncodedLen`)
 	/// Storage: `Nfts::AccountBalance` (r:1 w:1)
-	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(72), added: 2547, mode: `MaxEncodedLen`)
-	/// Storage: `Nfts::ItemConfigOf` (r:1 w:1)
-	/// Proof: `Nfts::ItemConfigOf` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
+	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(120), added: 2595, mode: `MaxEncodedLen`)
 	/// Storage: `System::Account` (r:1 w:1)
 	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
```

Comment:
FYI - Another useful source for checking the sizing of deposit bytes are these benchmark results.

Reply:
Hmm I checked the `ItemMetadata`'s max size and the configured bytes of `MetadataDeposit` in both testnet and devnet and it is different. Am I missing anything?

Reply:
Thank you!